panacea/query_client.go

```diff
@@ -160,24 +163,18 @@ func newVerifiedQueryClientWithDB(ctx context.Context, config *config.Config, in
 		return nil, err
 	}
 
-	// call refresh every minute
-	go func() {
-		for {
-			time.Sleep(1 * time.Minute)
-			if err := refresh(ctx, lc, trustedPeriod, &lcMutex); err != nil {
```

Comment:
You can remove the `refresh` function below.

Reply:
OK. I deleted it!(https://github.com/medibloc/panacea-oracle/pull/101/commits/a1abb012f74e92c8f24cf2d234873eb64d2fb094)